src/sentry/utils/tag_normalization.py

```diff
@@ -0,0 +1,89 @@
+import functools
+import re
+
+_KNOWN_TAGS = {
+    "sentry.cocoa",
+    "sentry.dart",
+    "sentry.dotnet",
+    "sentry.elixir",
+    "sentry.go",
+    "sentry.java",
+    "sentry.javascript.angular",
+    "sentry.javascript.browser",
+    "sentry.javascript.capacitor",
+    "sentry.javascript.cordova",
+    "sentry.javascript.deno",
+    "sentry.javascript.electron",
+    "sentry.javascript.ember",
+    "sentry.javascript.gatsby",
+    "sentry.javascript.miniapp",
+    "sentry.javascript.nextjs",
+    "sentry.javascript.node",
+    "sentry.javascript.node.experimental",
+    "sentry.javascript.node.hapi",
```

Comment:
can any of `sentry.javascript.node.*` be merged?

Reply:
It's a reasonable question. Given that there's only three of them, and they're still JS, I'd lean towards keeping the detail, though it might be worth figuring out where the experimental version is coming from, and whether or not that's us.

Reply:
I take it back - the Hapi SDK is a community one: https://github.com/hydra-newmedia/hapi-sentry.

Reply:
`node.experimental` does turn out to be us, though, and I think we should collapse it into the main node SDK.

https://github.com/getsentry/sentry-javascript/tree/170dcc571ac617b5c55a3522cc77a3f868c11e8d/packages/node-experimental